File: Graphics/Aditya_Rajput/1.py
nx = 200
ny = 100
with open('1.ppm', 'w') as f:
    print('P3\n{0} {1}\n255\n'.format(nx, ny), file = f)
    for j in range(ny-1, -1, -1):
        for i in range(0, nx, 1):
            print('{0} {1} {2}'.format(255.99 * i / nx, 255.99 * j / ny, 255.99 * 0.2), file = f)

print('END')

File: Graphics/Sarthak_Gupta/1.cpp
#include<iostream>
#include<fstream>
#include<cstdio>

int frac_to_rgb(float i){
	int f = int(255.99*i);
//	if(f>255) f = 255;
//	if(f<0) f = 0;
	return f;
}

using namespace std;

int main(){
	int npx = 200;
	int npy = 100;
	ofstream ofs("test.ppm", ios::out | ios::app);
	ofs<<"P3 \n"<<npx<<" "<<npy<<"\n255"<<endl;
	for(int j = 0 ; j<npy ; j++ ){
		for(int i = 0 ; i<npx ; i++){
			float b = float(i)/float(npx);
			float r = 0.9;
			float g = float(j)/float(npy);
			ofs<<frac_to_rgb(r) << " " << frac_to_rgb(g) << " " << frac_to_rgb(b)<<endl;
		}
	}
}

/* Played around with the loops to see what and how i can influence the actal image output,8 
*/

